js/src/relay/WalletLinkRelay.ts

```diff
@@ -593,10 +591,36 @@ export class WalletLinkRelay extends WalletLinkRelayAbstract {
         }
 
         WalletLinkRelay.accountRequestCallbackIds.add(id)
-      } else if (
-        request.method === Web3Method.switchEthereumChain ||
-        request.method === Web3Method.addEthereumChain
-      ) {
+      } else if (request.method === Web3Method.addEthereumChain) {
+        const cancel = () => {
+          this.handleWeb3ResponseMessage(
+            Web3ResponseMessage({
+              id,
+              response: AddEthereumChainResponse({ isApproved: false, rpcUrl: "" })
+            })
+          )
+        }
```

Comment:
nit: new line after 602 to be consistent with internal style guide

Reply:
added